testsuite/model/src/test/java/org/keycloak/testsuite/model/session/OfflineSessionPersistenceTest.java

```diff
@@ -258,6 +258,7 @@ public void testPersistenceMultipleNodesClientSessionsAtRandomNode() throws Inte
                     // on-going transaction, so it does not accept new invocations."
                     // also: org.infinispan.commons.CacheException: java.lang.IllegalStateException: Read commands must ignore leavers
                     if ((ex.getCause() != null && ex.getCause().getMessage().contains("ISPN000324") ||
+                            ex.getCause().getMessage().contains("ISPN000217") ||
```

Comment:
If you would want to keep this check, this might lead to a NPE as the `ex.getCause() != null` doesn't apply to the newly added condition. The current code relies on the `&&` vs. `||` operator precedence. Additional parentheses might make this more clear.

There is also an additional pair of parenthesis that is unnecessary.

Reply:
Good catch. I fixed the condition.